squirrel/driver/store_driver.py

```diff
@@ -19,17 +19,21 @@ class StoreDriver(MapDriver):
 
     name = "store_driver"
 
-    def __init__(self, url: str, serializer: SquirrelSerializer, **kwargs) -> None:
+    def __init__(
+        self, url: str, serializer: SquirrelSerializer, storage_options: Optional[Dict[str, Any]] = None, **kwargs
+    ) -> None:
         """Initializes StoreDriver.
 
         Args:
             url (str): the url of the store
             serializer (SquirrelSerializer): serializer to be passed to SquirrelStore
+            storage_options (Optional[Dict[str, Any]]): a dict with keyword arguments passed to filesystem initializer
             **kwargs: Keyword arguments to pass to the super class initializer.
         """
         super().__init__(**kwargs)
         self.url = url
         self.serializer = serializer
+        self.storage_options = storage_options if storage_options is not None else {}
```

Comment:
Just to demonstrate the shorthand for this, do not need to apply
```suggestion
        self.storage_options = storage_options or {}
```